docs/wiki/development/building.md

```diff
@@ -44,6 +44,46 @@ sudo tar xvf cmake-3.17.5-Linux-x86_64.tar.gz -C /usr/local --strip 1
 git clone https://github.com/osquery/osquery
 cd osquery
 
+# Build osquery
+mkdir build; cd build
+cmake -DOSQUERY_TOOLCHAIN_SYSROOT=/usr/local/osquery-toolchain ..
+cmake --build . -j10 # where 10 is the number of parallel build jobs
+```
+## Linux (Ubuntu 18, aarch64/arm64)
```

Comment:
What do you think of just calling out the differences here, it seems like the only changes are to build CMake from source and install the `aarch64` version of the toolchain.

Reply:
Also heads up that you want to include an empty newline between Markdown components, so a triple-backtick needs double newline before a heading. (tldr: insert another newline between lines 51 and 52).

Reply:
I'm happy to change it around to define `ARCH=$(uname -m)` use $ARCH in place of the x86_64/aarch64 downloads and then call out specifically the CMake difference. Building from source will work on x86_64 too, so the instructions could be 100% uniform with that extra step for x86.  Do you know if any future versions have fixed the RPM building issue, if I can use a newer version CMake, the binary releases would also support both platforms. 

Ack on the empty line.